src/common.build.pre.props

```diff
@@ -105,6 +105,7 @@
       <ProgramDatabaseFile>$(OutDir)$(TargetName)FullPDB.pdb</ProgramDatabaseFile>
       <SubSystem>Windows</SubSystem>
       <SetChecksum>true</SetChecksum>
+      <GenerateDebugInformation>DebugFull</GenerateDebugInformation>

```

Comment:
Neat. What's this do for us?

Reply:
I was hoping to generate a full PDB that SizeBench can analyze, but so far every build incantation I have devised keeps generating "/LTCG:incremental" which means SizeBench can't look at the PDBs.  I could revert this bit, but I'd prefer to find a way for at-least-release bits to stop using incremental LTCG so the PDBs are amenable to static analysis.

Reply:
Mind making this part of a separate changeset?  I'd like to go through and fix the damage that made us use the `FullPDB` suffix, as well, so it might be nice to put a little bow on it and call it an independent change

Reply:
Sure, no problem - I'll fix this when I figure out line endings, to just do one more push hopefully.